src/inet/InetInterface.cpp

```diff
@@ -502,6 +502,142 @@ void CloseIOCTLSocket()
     }
 }
 
+#if __ANDROID__
+
+static struct if_nameindex * backport_if_nameindex(void);
+static void backport_if_freenameindex(struct if_nameindex *);
+
+static void backport_if_freenameindex(struct if_nameindex * inArray)
+{
+    if (inArray == NULL)
+    {
+        return;
+    }
+
+    for (size_t i = 0; inArray[i].if_index != 0; i++)
+    {
+        if (inArray[i].if_name != NULL)
+        {
+            Platform::MemoryFree(inArray[i].if_name);
```

Comment:
set inArray[i].if_name = nullptr?

Reply:
The previous code was set to `NULL`, so it was added like that. Let's change it to `nullptr`.